prime-router/src/main/resources/metadata/hl7_mapping/receivers/Flexion/TIResults.yml

```diff
@@ -0,0 +1 @@
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
```

Comment:
Why is this file needed, if all it's doing is extending  ORU_R01-base? Why not use use ORU_R01 in the receiver settings?

Reply:
For now it's not used but we may need it in the future in case our ORU mapping diverges. It's following the convention we have for Orders `prime-router/src/main/resources/metadata/hl7_mapping/receivers/Flexion/TILabOrder.yml`. Having said that, I don't mind removing this for now and adding it back once we need it